pkg/chains/storage/docdb/docdb.go

```diff
@@ -179,11 +172,21 @@ func WatchBackend(ctx context.Context, cfg config.Config, watcherStop chan bool)
 		}
 	}()
 
-	// Add a path.
-	err = watcher.Add(cfg.Storage.DocDB.MongoServerURLDir)
-	if err != nil {
-		return nil, err
+	if cfg.Storage.DocDB.MongoServerURLPath != "" {
```

Comment:
Can we leverage `pathsToWatch` here instead?

Reply:
Good catch, thanks 👍🏻